tests/autoyast/console.pm

```diff
@@ -19,6 +19,8 @@ use testapi;
 
 sub run {
     my $self = shift;
+    # the main motivation for doing this test was bug#889757 and related
```

Comment:
can you be more specific what "bug#" means? Common reference is "bsc#" (bugzilla.suse.com) or "boo#" (bugzilla.opensuse.org)